### PR TITLE
Clean dexcom history

### PR DESCRIPTION
**Pull request by channemann**
Functional for dexcom glucose histories; specifically, handles the different entry keys ('glucose' instead of 'sgv' or 'amount' and 'display_time' instead of 'date') as well as throws out low values that indicate errors.

### CODE REVIEW

**Comment by channemann** [comment]
Added unit test for dex. I wasn't entirely certain how to split it until multiple unit tests, so it still runs as the same one with two sections. Additionally, renamed the original test fixutre to iter_glucose_mm.json to differentiate.

File: tests/glucose_tests.py [hunk 1]
```diff
@@ -11,7 +11,9 @@ def get_file_at_path(path):
 
 class CleanTestCase(unittest.TestCase):
     def test_clean_values(self):
```
Comment (loudnate) [inline tests/glucose_tests.py]: Let's make two separate functions instead: `test_clean_values_mm` and `test_clean_values_dex`

File: openapscontrib/mmglucosetools/glucose.py [hunk 2]
```diff
@@ -17,5 +19,9 @@ def clean(glucose_entries):
     :rtype: list(dict)
     """
     cleaned = filter(_filter_entry, glucose_entries)
-
-    return sorted(cleaned, key=lambda x: x['date'], reverse=True)
```
Comment (loudnate) [inline openapscontrib/mmglucosetools/glucose.py:21]: You can actually turn this into one line:

``` python
return sorted(cleaned, key=lambda x: x.get('date', x.get('display_time')), reverse=True)
```

**Comment by channemann** [comment]
Changes implemented, passed both tests on my Pi:

```
pi@raspberrypi ~/openaps-mmglucosetools $ sudo python setup.py test
running test
running egg_info
writing requirements to openapscontrib.mmglucosetools.egg-info/requires.txt
writing openapscontrib.mmglucosetools.egg-info/PKG-INFO
writing namespace_packages to openapscontrib.mmglucosetools.egg-info/namespace_packages.txt
writing top-level names to openapscontrib.mmglucosetools.egg-info/top_level.txt
writing dependency_links to openapscontrib.mmglucosetools.egg-info/dependency_links.txt
writing requirements to openapscontrib.mmglucosetools.egg-info/requires.txt
writing openapscontrib.mmglucosetools.egg-info/PKG-INFO
writing namespace_packages to openapscontrib.mmglucosetools.egg-info/namespace_packages.txt
writing top-level names to openapscontrib.mmglucosetools.egg-info/top_level.txt
writing dependency_links to openapscontrib.mmglucosetools.egg-info/dependency_links.txt
reading manifest file 'openapscontrib.mmglucosetools.egg-info/SOURCES.txt'
writing manifest file 'openapscontrib.mmglucosetools.egg-info/SOURCES.txt'
running build_ext
test_clean_values_dex (tests.glucose_tests.CleanTestCase) ... ok
test_clean_values_mm (tests.glucose_tests.CleanTestCase) ... ok

----------------------------------------------------------------------
Ran 2 tests in 0.016s

OK
```

**Comment by loudnate** [comment]
Nice work!

**Pull request merged**